includes/group-sites.php

```diff
@@ -1820,3 +1856,145 @@ function cboxol_load_theme_specific_fixes() {
 	}
 }
 add_action( 'wp_enqueue_scripts', 'cboxol_load_theme_specific_fixes' );
+
+/**
+ * Generate array of nav menu items.
+ *
+ * @return array $items
+ */
+function cboxol_get_nav_menu_items() {
+	$items = [];
+
+	$group_id = openlab_get_group_id_by_blog_id( get_current_blog_id() );
+	if ( ! $group_id ) {
+		return $items;
+	}
+
+	$group = groups_get_group( $group_id );
+	if ( ! $group->is_visible ) {
+		return $items;
+	}
+
+	$group_type = cboxol_get_group_group_type( $group_id );
+	if ( ! is_wp_error( $group_type ) ) {
```

Comment:
I guess only '[Group Profile]' needs to be added in this way, because the `title` is dynamic based on group type. That's why the `CBOX OpenLab` menu only has one link under it. 'Home' is just a regular custom link. Is that reasoning correct?

Reply:
That is correct.